### PR TITLE
proftpd: bump to 1.3.6 (solaris-userland)

### PR DESCRIPTION
**Pull request by alarcher**
No idea how to test this...

### CODE REVIEW

File: components/network/proftpd/Makefile [hunk 1]
```diff
@@ -20,45 +20,47 @@
...
 
+BUILD_BITS=64
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		proftpd
-COMPONENT_VERSION=	1.3.5
-COMPONENT_REVISION=	5
-COMPONENT_SUBVERSION=	d
+COMPONENT_VERSION=	1.3.6
+COMPONENT_REVISION=	6
```
Comment (pyhalov) [inline components/network/proftpd/Makefile]: You can remove it as COMPONENT_VERSION increased.

File: components/network/proftpd/proftpd.pam [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#
+# Copyright (c) 2019, Oracle and/or its affiliates. All rights reserved.
+#
+# PAM stack customized for FTP comes without pam_fm_notify.
+#
+session definitive	pam_user_policy.so.1
```
Comment (pyhalov) [inline components/network/proftpd/proftpd.pam]: We don't have pam_user_policy or pam_fm_notify. So, this makes session case default one, and we don't need this customized file.
BTW, I'm not sure it works as desired anyway.

**Review by pyhalov — CHANGES_REQUESTED** [review]

File: components/network/proftpd/Makefile (hunk 1 above)
Comment (alarcher) [reply components/network/proftpd/Makefile]: Fixed.

File: components/network/proftpd/proftpd.pam (hunk 2 above)
Comment (alarcher) [reply components/network/proftpd/proftpd.pam]: Removed.

**Comment by pyhalov** [comment]
I didn't have time to look at it attentively, but I'd say push it if it works for you.

**Pull request merged**